assets/texts/fr/amoSummary.txt

```diff
@@ -1,2 +1,3 @@
-Ce module complémentaire vous donne un bon petit sélecteur d'émoticônes que vous pouvez pour en trouver 🔎 et en sélectionner. 😀
-Il sera ajouté à votre presse-papier 📋 (de façon similaire à un "copier-coller") quand vous cliquez 🖱️ dessus.
+Ce module complémentaire vous donne un sélecteur d'émoji moderne pour en trouver 🔎 et les utiliser. 😀 Il sera ajouté à votre presse-papier 📋 (Copie / 'CTRL-C'), ou inséré directement, quand vous cliquez 🖱️ dessus.
```

Comment:
> I mean, is there no well-known term for "copy to clipboard"? It's the same term French computer users should know from their context menu when selecting text or e.g. copying files in their file manager.

Well, there is one term for it, but it's a bit uncommon, to most people; and simply saying "copy" might feel a bit off.
I will change it to "copy-paste".

>(Ugh, I see that is not even translated…)
👀

Reply:
Then i wonder what the menu entries in Firefox, your file manager etc. say?

Reply:
To be honest, I'm not even sure there is an used translation for it, I guess it would be a "copy into clipboard" I think.

But with technophobes, it might totally unknown to them.